games/rust/entrypoint.sh

```diff
@@ -11,17 +11,36 @@ export INTERNAL_IP=`ip route get 1 | awk '{print $(NF-2);exit}'`
 MODIFIED_STARTUP=`eval echo $(echo ${STARTUP} | sed -e 's/{{/${/g' -e 's/}}/}/g')`
 echo ":/home/container$ ${MODIFIED_STARTUP}"
 
-# OxideMod has been replaced with uMod
-if [ -f OXIDE_FLAG ] || [ "${OXIDE}" = 1 ] || [ "${UMOD}" = 1 ]; then
+if [ -f CARBON_FLAG ] || [ "${FRAMEWORK}" == "carbon" ]; then
+    # Carbon: https://github.com/CarbonCommunity/Carbon.Core
+    echo "Updating Carbon..."
+    curl -sSL "https://github.com/CarbonCommunity/Carbon.Core/releases/download/production_build/Carbon.Linux.Release.tar.gz" | tar zx
+    echo "Done updating Carbon!"
+
+    # backward compatibility
+    export CARBON=1
+
+    export DOORSTOP_ENABLED=1
+    export DOORSTOP_TARGET_ASSEMBLY="$(pwd)/carbon/managed/Carbon.Preloader.dll"
+    MODIFIED_STARTUP="LD_PRELOAD=$(pwd)/libdoorstop.so ${MODIFIED_STARTUP}"
+
+elif [ -f OXIDE_FLAG ] || [ "${FRAMEWORK}" == "oxide" ]; then
```

Comment:
```suggestion
elif [[ "$OXIDE" == "1" ]] || [[ "${FRAMEWORK}" == "oxide" ]]; then
    echo "Updating uMod..."
```

You'd want to check for `OXIDE` string being 1 as that's what the default egg is using. I don't see any reason to check for the existence of `OXIDE_FLAG` file (or CARBON_FLAG).